### ISSUE TITLE
how can I disable slide to open?

### ISSUE THREAD

**Issue by JungMinHur** [nativescript-vue/nativescript-vue-multi-drawer#8]

**Comment by rigor789** [comment]
You can set `canSwipeOpen` to `false` to disable it:

https://github.com/nativescript-vue/nativescript-vue-multi-drawer/blob/952d86a23ab79713e3aef1bd145bcdfb25dc8f64/index.js#L12

For example to disable swipe to open for the left drawer:
```js
// in the drawer config
{
  left: {
    canSwipeOpen: false
  }
}
```

**Comment by JungMinHur** [comment]
Thanks for your apply!!

is there a way to disable all swipe actions? not just open

**Comment by rigor789** [comment]
What do you mean exactly? Completely disable user interaction?

**Comment by JungMinHur** [comment]
yes I mean, completely block swipe actions to open and close

**Comment by gravataLonga** [comment]
as far i can tell, you must provider false to each side. `{left: {canSwipeOpen: false}, top: {canSwipeOpen: false}, right: {canSwipeOpen: false}, bottom: {canSwipeOpen: false}}`

**Comment by JungMinHur** [comment]
yep! I know how to disable swipe open, but I want to know how to disable close too. It seems that even if I set `{canSwipeOpen: false}` swipe to close is still working!

**Comment by rigor789** [comment]
There's no way right now - but you could easily implement a static "drawer" with GridLayout+`animate` yourself, using this is pointless if you want to disable everything.
